pyp2rpm/metadata_extractors.py

```diff
@@ -134,7 +134,8 @@ def inner(self):
 
         temp_dir = tempfile.mkdtemp()
         try:
-            extractor = virtualenv.VirtualEnv(self.name, temp_dir,
+            extractor = virtualenv.VirtualEnv(self.local_file, self.version,
+                                              temp_dir,
```

Comment:
Is there a change from the "reuse local files" PR in here?
Otherwise I don't understand the `self.name` → `self.local_file` change.
And I'm not familiar with virtualenv, but if those are positional arguments, this will blow up :)

Reply:
Right.  I'd introduced the "version" arg in this branch, so this was a result of resolving the merge conflict with the parent branch during a rebase.